bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/AgsClient.scala

```diff
@@ -1,245 +0,0 @@
-      font       = font.deriveFont(Font.BOLD).deriveFont(font.getSize2D + 8.0f)
-      foreground = Color.BLACK
-    }
-    layout(statusMessage) = new Constraints {
-      gridx   = 1
-      gridy   = 0
-      fill    = Horizontal
-      weightx = 1.0
-      anchor  = West
-    }
-
-    val explanatoryText = new TextArea() {
-      wordWrap      = true
-      lineWrap      = true
-      editable      = false
-      preferredSize = new Dimension(0, 50)
-      foreground    = Color.DARK_GRAY
-      text          = Note
-    }
-    layout(explanatoryText) = new Constraints {
-      gridx     = 0
-      gridy     = 1
-      gridwidth = 2
-      fill      = Horizontal
-      weightx   = 1.0
-      insets    = new Insets(15, 10, 5, 10)
-    }
-
-    layout(new BorderPanel()) = new Constraints {
-      gridx     = 0
-      gridy     = 2
-      gridwidth = 2
-      fill      = Both
-      weightx   = 1.0
-      weighty   = 1.0
-    }
-
-    object buttonPanel extends GridBagPanel {
-      border = BorderFactory.createCompoundBorder(
-                 BorderFactory.createMatteBorder(1, 0, 0, 0, Color.LIGHT_GRAY),
-                 BorderFactory.createEmptyBorder(10, 0, 0, 0))
-
-      layout(new BorderPanel()) = new Constraints {
-        gridx   = 0
-        fill    = Horizontal
-        weightx = 1.0
-      }
-
-      val acceptButton = new Button("OK") {
-        enabled = false
-      }
-      layout(acceptButton) = new Constraints {
-        gridx = 1
-      }
-      val cancelButton = new Button("Cancel")
-      layout(cancelButton) = new Constraints {
-        gridx  = 2
-        insets = new Insets(0, 5, 0, 0)
-      }
-    }
-
-    layout(buttonPanel) = new Constraints {
-      gridx     = 0
-      gridy     = 4
-      gridwidth = 2
-      fill      = Horizontal
-      weightx   = 1.0
-    }
-  }
-
-  contents = mainPanel
-
-  import mainPanel.buttonPanel.cancelButton
-  listenTo(cancelButton)
-  reactions += {
-    case ButtonClicked(`cancelButton`) => quit()
-  }
-
-  private def quit(): Unit = {
-    cancelled = true
-    visible   = false
-    dispose()
-  }
-
-  private def handleSuccess(selOpt: Option[AgsStrategy.Selection]): Unit =
-    if (!cancelled) {
-      selOpt match {
-        case Some(sel) =>
-          applySelection(sel)
-          showTpeFeatures(sel)
-          quit()
-        case _ =>
-          setupManualSearch(NothingFoundMessage)
-      }
-    }
-
-  private def setupManualSearch(msg: String): Unit = {
-    mainPanel.explanatoryText.text = msg + "  Try changing the conditions, moving the field, or doing a manual search."
-    mainPanel.statusIcon.icon      = FailureIcon
-    mainPanel.statusMessage.text   = "No guide star was found."
-
-    import mainPanel.buttonPanel.acceptButton
-    acceptButton.text = "Manual Search"
-    acceptButton.enabled = true
-    listenTo(acceptButton)
-    reactions += {
-      case ButtonClicked(`acceptButton`) =>
-        TpeGuideStarDialog.showDialog(TpeManager.open().getImageWidget)
```

Comment:
I am wondering why it has been deprecated, though. Any ideas?
Is there another way to do this?
I just removed the no longer used `TpeGuideStarDialogInternalFrame` (I suspect this class has been redundant for some time).